### DISCUSSION TITLE
Editing paper.yml via environment variables?

### DISCUSSION THREAD

**Discussion by Tamariniak** [itzg/docker-minecraft-server#973] [Q&A]
I'm running a Paper server, deployed via Portainer Stacks, and I need to change some settings in the paper.yml document. It's this one specifically:

`settings:
  unsupported-settings:
    allow-permanent-block-break-exploits: true`

I've tried setting it as an environment variable in all ways I could think of, but nothing seems to work. How should I go about this?

**Comment by itzg** [comment]
This image doesn't manage paper configs. Edit the files yourself in the data volume.
